scipy-1.0/scipy-optimize.tex

```diff
@@ -130,3 +130,7 @@ \subsubsection*{Nonlinear Optimization}
   \end{tabular}
   \label{tab:minimize}
 \end{table}
+
+% Some problems may require the use of global optimizers to find the minimum of the objective function over the set of all parameter values, especially if the objective function possesses many local minima that other solvers could get stuck in.
+% \texttt{Basinhopping} \cite{Wales1997} uses random perturbation of the parameter vector, with local minimization. The acceptance of the new coordinates is dictated by the Metropolis test typically used in Monte Carlo simulations.
+% \texttt{differential\textunderscore evolution} uses
```

Comment:
is this an incomplete comment / thought about `differential_evolution`?

Reply:
It's not an incomplete thought. I wasn't sure if there was going to be space in the article, so I left it commented out. I'm happy if it is added to the manuscript.

Reply:
It looks like it's missing half the text on DE!

Reply:
In fact it's missing half the text I wrote. Not sure what went on with my text editor. I'll have to rewrite the bits.